src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer" public="false">
```

Comment:
How can we handle the case that someones uses the FrameworkBundle with an older version of the Serializer component that doesn't contain this class?